### ISSUE TITLE
Can't view file content on win10 !

### ISSUE THREAD

**Issue by DFVSQY** [DonJayamanne/gitHistoryVSCode#278]
when i view file content on win10 by `Git: view file history` and `View file contents`, it failed,  log is below:
~~~
git rev-parse --show-toplevel  (completed in 0.001s)
{"stack":"Error: spawn F:/Program Files/Git/bin/git.exe ENOENT\n\tat exports._errnoException (util.js:1050:11)\n\tat Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)\n\tat onErrorNT (internal/child_process.js:367:16)\n\tat _combinedTickCallback (internal/process/next_tick.js:80:11)\n\tat process._tickCallback (internal/process/next_tick.js:104:9)","message":"spawn F:/Program Files/Git/bin/git.exe ENOENT","code":"ENOENT","errno":"ENOENT","syscall":"spawn F:/Program Files/Git/bin/git.exe","path":"F:/Program Files/Git/bin/git.exe","spawnargs":["rev-parse","--show-toplevel"]}
git rev-parse --show-toplevel  (completed in 0.002s)
~~~

**Comment by DonJayamanne** [comment]
Please confirm the following file exists?
`F:/Program Files/Git/bin/git.exe`

**Comment by JWleo** [comment]
same problem on my win10,and i'm sure the path is right
![image](https://user-images.githubusercontent.com/32316654/40285864-8897499a-5cd3-11e8-968b-ee2901b72532.png)

**Comment by DFVSQY** [comment]
@DonJayamanne  yes, it does exist.

**Comment by izderadicka** [comment]
I actually do have similar problem on linux. "View file contents" does nothing. But there is also no output in output console of the plugin. Like if I did not click anything.

**Comment by Sevyls** [comment]
same as #283

**Comment by WarrenFerrell** [comment]
Having the same problem. path is also correct. The directory exists and `git rev-parse --show-toplevel` works by itself.  My repository is on a separate drive tho : S/...

```bash
git rev-parse --show-toplevel  (completed in 0.001s)
{"stack":"Error: spawn C:/Program Files/Git/bin/git ENOENT\n    at exports._errnoException (util.js:1050:11)\n    at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)\n    at onErrorNT (internal/child_process.js:367:16)\n    at _combinedTickCallback (internal/process/next_tick.js:80:11)\n    at process._tickCallback (internal/process/next_tick.js:104:9)\n","message":"spawn C:/Program Files/Git/bin/git ENOENT","code":"ENOENT","errno":"ENOENT","syscall":"spawn C:/Program Files/Git/bin/git","path":"C:/Program Files/Git/bin/git","spawnargs":["rev-parse","--show-toplevel"]}
git rev-parse --show-toplevel  (completed in 0.002s)
````

**Comment by sonnyandersson** [comment]
Same issue since last VS Code update

**Comment by liximomo** [comment]
I've made a pr for this. Hope @DonJayamanne can merge it soon.

**Comment by ole1986** [comment]
Is this still an issues in version 0.4.4?

**Comment by ole1986** [comment]
@liximomo since your PR is not yet implemented but we are already in version 0.4.6 I am asking myself if this is still an issue? Can you somehow check this.

If this is fixed we can close your PR without merging
Thank you
